cpp/velox/compute/VeloxPlanConverter.cc

```diff
@@ -88,8 +92,39 @@ void VeloxInitializer::Init() {
   }
   auto hdfsPort = hdfsUri.substr(hdfsUri.find(":") + 1);
   auto hdfsHost = hdfsUri.substr(0, hdfsUri.find(":"));
-  static const std::unordered_map<std::string, std::string> configurationValues(
+  static const std::unordered_map<std::string, std::string> hdfsConfig(
       {{"hive.hdfs.host", hdfsHost}, {"hive.hdfs.port", hdfsPort}});
+  configurationValues.merge(hdfsConfig);
+#endif
+
+#ifdef VELOX_ENABLE_S3
+  filesystems::registerS3FileSystem();
+  // TODO(yuan): should passing thru config?
+  std::string awsAccessKey = "minio";
```

Comment:
default value doesn't seem to be used. Maybe keep it as empty string for consistency, and easy to validate in code?